src/main.rs

```diff
@@ -75,21 +98,28 @@ fn main() {
 
     let format = value_t!(matches, "format", Format).unwrap_or_else(|e| e.exit());
 
-    let output = Command::new("lctl")
-        .arg("get_param")
-        .args(parser::params())
+    let handle = thread::spawn(move || -> Result<Vec<Record>> {
+        let lctl_output = get_lctl_output()?;
+        let lctl_record = parse_lctl_output(&lctl_output)?;
+
+        Ok(lctl_record)
+    });
+
+    let lnetctl_output = Command::new("lnetctl")
+        .arg("export")
         .output()
-        .expect("failed to get lctl stats");
+        .expect("failed to get lnetctl stats");
 
-    let stats = str::from_utf8(&output.stdout).expect("while converting stdout from utf8");
+    let lnetctl_stats =
+        str::from_utf8(&lnetctl_output.stdout).expect("while converting lnetctl stdout from utf8");
 
-    let (record, state) = parser::parse()
-        .easy_parse(State::new(stats))
-        .expect("while parsing stats");
+    let lnet_record = lnetctl_parser::parse(lnetctl_stats).expect("while parsing lnetctl stats");
 
-    if state.input != "" {
-        eprintln!("Content left in input buffer: {}", state.input)
-    }
+    let lctl_record = handle.join().unwrap().unwrap();
+
+    let mut record = vec![];
```

Comment:
Any reason not to do:
```
let mut record = vec![lctl_record, lnet_record];
```
?

Reply:
Not that I can see. I'll fix it in the next patch.